### ISSUE TITLE
Error with shap values in 0.3.8.1

### ISSUE THREAD

**Issue by tvdboom** [oegedijk/explainerdashboard#202]
Hi,
since the new update I am getting the following error:

```
venv310\lib\site-packages\explainerdashboard\explainers.py:2396: in set_shap_values
    self._shap_values_df = pd.concat([self._shap_values_df,
venv310\lib\site-packages\pandas\util\_decorators.py:311: in wrapper
    return func(*args, **kwargs)
venv310\lib\site-packages\pandas\core\reshape\concat.py:346: in concat
    op = _Concatenator(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pandas.core.reshape.concat._Concatenator object at 0x000001770C6331F0>
objs = [[],      mean radius  mean texture  ...  worst symmetry  worst fractal dimension
0            1.0      1.103333  ... ...             1.0
399          1.0      0.948333  ...             1.0                      1.0

[400 rows x 30 columns]]
axis = 0, join = 'outer', keys = None, levels = None, names = None
ignore_index = False, verify_integrity = False, copy = True, sort = False

    def __init__(
        self,
        objs: Iterable[NDFrame] | Mapping[Hashable, NDFrame],
        axis=0,
        join: str = "outer",
        keys=None,
        levels=None,
        names=None,
        ignore_index: bool = False,
        verify_integrity: bool = False,
        copy: bool = True,
        sort=False,
    ):
        if isinstance(objs, (ABCSeries, ABCDataFrame, str)):
            raise TypeError(
                "first argument must be an iterable of pandas "
                f'objects, you passed an object of type "{type(objs).__name__}"'
            )
    
        if join == "outer":
            self.intersect = False
        elif join == "inner":
            self.intersect = True
        else:  # pragma: no cover
            raise ValueError(
                "Only can inner (intersect) or outer (union) join the other axis"
            )
    
        if isinstance(objs, abc.Mapping):
            if keys is None:
                keys = list(objs.keys())
            objs = [objs[k] for k in keys]
        else:
            objs = list(objs)
    
        if len(objs) == 0:
            raise ValueError("No objects to concatenate")
    
        if keys is None:
            objs = list(com.not_none(*objs))
        else:
            # #1649
            clean_keys = []
            clean_objs = []
            for k, v in zip(keys, objs):
                if v is None:
                    continue
                clean_keys.append(k)
                clean_objs.append(v)
            objs = clean_objs
    
            if isinstance(keys, MultiIndex):
                # TODO: retain levels?
                keys = type(keys).from_tuples(clean_keys, names=keys.names)
            else:
                name = getattr(keys, "name", None)
                keys = Index(clean_keys, name=name)
    
        if len(objs) == 0:
            raise ValueError("All objects passed were None")
    
        # figure out what our result ndim is going to be
        ndims = set()
        for obj in objs:
            if not isinstance(obj, (ABCSeries, ABCDataFrame)):
                msg = (
                    f"cannot concatenate object of type '{type(obj)}'; "
                    "only Series and DataFrame objs are valid"
                )
>               raise TypeError(msg)
E               TypeError: cannot concatenate object of type '<class 'list'>'; only Series and DataFrame objs are valid

venv310\lib\site-packages\pandas\core\reshape\concat.py:436: TypeError
```

The error is produced here:

https://github.com/oegedijk/explainerdashboard/blob/518cefca18d8a8c217709395105719618418942f/explainerdashboard/explainers.py#L2379-L2396

self._shap_values_df is defined as list and never turned to a pandas object before the concatenation. It could be that this work with an older version of pandas, I am running 1.4.1

**Comment by oegedijk** [comment]
Ah, I can see what happened there: a recent PR substituted all the recently deprecated `pd.append()` calls with `pd.concat` , however in this case it was actually a list that was getting appended, so no need to substitute :)

I missed it in code review, and I now realize that there was also a missing test for this method :(

Anyway, should be easy to reverse!

**Comment by oegedijk** [comment]
okay, just released the fix (0.3.8.2), let me know if it fixes the issue for you.

**Comment by tvdboom** [comment]
yes, it works. thanks for the quick hotfix!
